forum/qa-plugin/report-reason/script.js

```diff
@@ -1,42 +1,31 @@
-$(document).ready(function()
-{
-    $(".qa-form-light-button-flag").attr("type", "button");
+document.addEventListener("DOMContentLoaded", () => {
     
-    $(".qa-form-light-button-flag").click( function()
-    {
-        var postId = $(this).data("postid");
-        var postType = $(this).data("posttype");
-        var parentId = $(this).data("parentid");
+    const flagButton = document.getElementsByClassName('qa-form-light-button-flag')[0];
```

Comment:
Zamiast `getElementsByClassName` i potem wybierać pierwszy element z listy, użyj po prostu `querySelector` - to samo dotyczy pozostałych przypadków.

Reply:
`document.querySelector('div.qa-form-light-button-flag');` ?